src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,6 +134,13 @@ public void deleteTask(Task toDelete) {
         tasks.remove(toDelete);
     }
 
+    @Override
+    public void setTask(Task target, Task editedTask) {
+        requireAllNonNull(target, editedTask);
+
+        tasks.setTask(target, editedTask);
+    }
+
```

Comment:
I might need to change these in the future since this should be addressbook.setTask() not tasks.setTask() but I believe its ok for now (I can just change it)